docs/quickstart-eks.md

```diff
@@ -0,0 +1,264 @@
+
+## Installation and setup
+
+### Step 1: Set up the cluster
+
+In this section, you will deploy an Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster using the `eksctl` tool`.
+
+* Make sure you have a working Amazon CLI environment. Please refer to [Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) if in doubt.
+
+* Configure the following environment variables:
+
+  ```bash
+  export BKPR_DNS_ZONE=my-domain.com
+  export AWS_EKS_USER=my-user@example.com
+  export AWS_EKS_CLUSTER=my-eks-cluster
+  export AWS_EKS_K8S_VERSION=1.11
+  ```
+
+  - `BKPR_DNS_ZONE` specifies the DNS suffix for the externally-visible websites and services deployed in the cluster.
+  - `AWS_EKS_USER` specifies the email address used in requests to Let's Encrypt.
+  - `AWS_EKS_CLUSTER` specifies the name of the AKS cluster.
+  - `AWS_EKS_K8S_VERSION` specifies the version of Kubernetes to use for creating the cluster. The [BKPR Kubernetes version support matrix](../README.md#kubernetes-version-support-matrix-for-bkpr-10) lists the base Kubernetes versions supported by BKPR.
+
+* Create the EKS cluster:
+
+  ```bash
+  eksctl create cluster --name=${AWS_EKS_CLUSTER} \
+                        --color=fabulous \
+                        --nodes=3 \
+                        --version=${AWS_EKS_K8S_VERSION}
+  ```
+
+  Provisioning an EKS cluster can take a long time to complete. Please be patient while the request is being processed.
+
+* Configure `kubectl` to use the new cluster:
+
+  ```bash
+  aws eks update-kubeconfig --name=${AWS_EKS_CLUSTER}
+  ```
+
+* Verify that your cluster is up and running:
+
+  ```bash
+  kubectl get nodes
+  ```
+
+### Step 2: Configure a User Pool in AWS Cognito
+
+In order to authenticate users for applications running atop EKS, BKPR requires a User Pool in AWS Cognito to be configured.
+
+If you already have a working User Pool in AWS Cognito that you would like to use for authenticating users, you will need to retrieve its ID in the form `${AWS_REGION}-${USER_POOL_ID}`, and write it down. Using this `Pool Id`, skip at the end of this section.
+
+Else, If you are new to using BKPR on EKS, or if you want to create a new User Pool in AWS Cognito, please follow these steps:
+
+1. Browse to the [Cognito](https://console.aws.amazon.com/cognito/) module in the AWS Console
+2. Click on the `Manage User Pools` button
+3. Click on the `Create a user pool` button on the top-right corner
+4. Enter a valid `Pool name` like `eks-test`
+5. Click on the `Step through settings` button
+6. In the `Attributes` section leave all the checkboxes unticked as shown below:
+
+<p align="center"><img src="eks/1-Attributes.png" width=840/></p>
+
+7. Click on the `Next step` button
```

Comment:
I think this step should be merged with the previous step as in we can write the previous step as follows:

In the **Attributes** section leave all the check-boxes unticked and go to the **Next step**:

The same suggestion goes for other instances as well